feature/experimental/policy/policy_vrf_selection/otg_tests/multiple_vrfs_and_gue_decap/README.md

```diff
@@ -0,0 +1,132 @@
+# RT-3.3: Multiple VRFs and GUE DECAP in Default VRF
+
+## Summary
+This test ensures NOS is able to host multiple VRFs and at the sametime perform GUE tunnel DECAP in the default VRF
+
+## Procedure
+Test environment setup
+
+### Topology
+Create the following connections:
+```mermaid
+graph LR; 
+subgraph ATE1 [ATE1]
+    A1[Port1] 
+    A2[Port2]
+end
+
+subgraph DUT1 [DUT1]
+    B1[Port1]
+    B2[Port2]
+end
+
+subgraph ATE2 [ATE2]
+    C1[Port1] 
+    C2[Port2]
+end
+
+A1 <-- IBGP(ASN100) --> B1; 
+B2 <-- EBGP(ASN100:ASN200) --> C1;
+A2 <-- EBGP(ASN100:ASN200) --> C2;
+```
+
+### Advertisements:
+
+	ATE1:Port1 advertises following prefixes over IBGP to DUT:Port1
+		- IPv4Prefix1/24 IPv6Prefix1/64
+		- IPv4Prefix2/24 IPv6Prefix2/64
+		- IPv4Prefix3/24 IPv6Prefix3/64
+		- IPv4Prefix4/24 IPv6Prefix4/64
+		- IPv4Prefix5/24 IPv6Prefix5/64
+	
+	ATE2:Port1 advertieses following prefixes to DUT:Port2 over EBGP
+		- IPv4Prefix6/24 IPv6Prefix6/64
+		- IPv4Prefix7/24 IPv6Prefix7/64
+		- IPv4Prefix8/24 IPv6Prefix8/64
+		- IPv4Prefix9/24 IPv6Prefix9/64
+		- IPv4Prefix10/24 IPv6Prefix10/64
+
+	DUT:Port2 advertises following tunnel endpoint prefixe to ATE2:Port1 over EBGP. 
+		- IPv4Prefix12/28
+
+	ATE1:Port2 advertises following tunnel endpoint prefix to ATE2:Port2
+		- IPv4Prefix11/28
+
+### Flows:
+
+	From ATE1:Port1 to ATE2:Port1
+		- IPv4Prefix1/24 to IPv4Prefix6/24 [DSCP:BE1]
+		- IPv6Prefix1/64 to IPv6Prefix6/64 [DSCP:BE1]
+
+		- IPv4Prefix2/24 to IPv4Prefix7/24 [DSCP:AF1]
+		- IPv6Prefix2/64 to IPv6Prefix7/64 [DSCP:AF1]
+
+		- IPv4Prefix3/24 to IPv4Prefix8/24 [DSCP:AF2]
+		- IPv6Prefix3/64 to IPv6Prefix8/64 [DSCP:AF2]
+
+		- IPv4Prefix4/24 to IPv4Prefix9/24 [DSCP:AF3]
+		- IPv6Prefix4/64 to IPv6Prefix9/64 [DSCP:AF3]
+
+		- IPv4Prefix5/24 to IPv4Prefix10/24 [DSCP:AF4]
+		- IPv6Prefix5/64 to IPv6Prefix10/64 [DSCP:AF4]
+
+	Flows from ATE2:Port1 to ATE1:Port1
+		- IPv4Prefix6/24 to IPv4Prefix1/24 [DSCP:BE1]
+		- IPv6Prefix6/64 to IPv6Prefix1/64 [DSCP:BE1]
+
+		- IPv4Prefix7/24 to IPv4Prefix2/24 [DSCP:AF1]
+		- IPv6Prefix7/64 to IPv6Prefix2/64 [DSCP:AF1]
+
+		- IPv4Prefix8/24 to IPv4Prefix3/24 [DSCP:AF2]
+		- IPv6Prefix8/64 to IPv6Prefix3/64 [DSCP:AF2]
+
+		- IPv4Prefix9/24 to IPv4Prefix4/24 [DSCP:AF3]
+		- IPv6Prefix9/64 to IPv6Prefix4/64 [DSCP:AF3]
+
+		- IPv4Prefix10/24 to IPv4Prefix5/24 [DSCP:AF4]
+		- IPv6Prefix10/64 to IPv6Prefix5/64 [DSCP:AF4]
+
+  - DUT has 2 VRFs, Default and Non-Default. Ensure leaking routes from the default to the non-default VRFs so prefixes learnt over IBGP and EBGP are available in the tables of bothe VRFs.
```

Comment:
Maybe just worth clarifying (AIUI):

 * DUT port 1 and port 2 are assigned to the `DEFAULT` VRF.
 * BGP session between ATE1 and DUT1 is in the `DEFAULT` VRF.
 * BGP session between ATE2 and DUT2 is in the `DEFAULT` VRF.

So there are no routes learnt into the `Non-Default` VRF other than by importing them from the `DEFAULT` VRF. Right?